tests/test_transforms/test_decomposition/test_stl_transform.py

```diff
@@ -67,41 +88,56 @@ def ts_trend_seasonal_nan_tails() -> TSDataset:
 
 
 @pytest.mark.parametrize("model", ["arima", "holt"])
-def test_transform_one_segment(df_trend_seasonal_one_segment, model):
+@pytest.mark.parametrize(
+    "df_name", ["df_trend_seasonal_one_segment", "df_trend_seasonal_starting_with_nans_one_segment"]
+)
+def test_transform_one_segment(df_name, model, request):
     """Test that transform for one segment removes trend and seasonality."""
+    df = request.getfixturevalue(df_name)
     transform = _OneSegmentSTLTransform(in_column="target", period=7, model=model)
-    df_transformed = transform.fit_transform(df_trend_seasonal_one_segment)
-    np.testing.assert_allclose(df_transformed["target"], 0, atol=0.2)
+    df_transformed = transform.fit_transform(df)
+    df_expected = df.copy()
```

Comment:
We don't need copy I guess. We've already made new instance from fixtures for every test cases